### DISCUSSION TITLE
About the project

### DISCUSSION THREAD

**Discussion by joaovitorpmoreira** [tukaan/tukaan#148] [Feedback]
Hello! I would like to know if the project is still active.

**Comment by Moosems** [comment]
Sadly it isn't all that active as building a good UI framework off of Tk isn't all that great of an idea and the owner, rdbende, has been far too busy to maintain the project.

**Comment by rdbende** [comment]
I still quite like the project, and I think the Python community really needs something simple, like tkinter but written for the 21st century. I'd really like to see someone picking it up and continue developing it, however I'm no longer interested in it, as I found something that better suits my needs.
